jobserver/views.py

```diff
@@ -411,6 +411,17 @@ def get_queryset(self):
         return qs
 
 
+@method_decorator(login_required, name="dispatch")
+class WorkspaceNotificationsToggle(View):
+    def post(self, request, *args, **kwargs):
+        workspace = get_object_or_404(Workspace, name=self.kwargs["name"])
+
+        workspace.will_notify = not workspace.will_notify
```

Comment:
This is fine and I doubt will cause us any problems in real life, but I think I'd prefer doing the toggling logic to generate a hidden input in the form so we could guarantee that form's effect would always match the text on the button. (No need to make any changes now, just wanted to mention it.)